### DISCUSSION TITLE
Copy and Paste Written Lessons on Cyfrin platform

### DISCUSSION THREAD

**Discussion by pythondefichain** [Cyfrin/foundry-full-course-cu#2869] [Q&A]
How do I copy and paste the Written lessons on Cyfrin Updraft site?

**Comment by heheboii11** [comment]
<img width="728" alt="Screenshot 2024-11-15 at 11 39 01 AM" src="https://github.com/user-attachments/assets/920ddd7e-7bc4-48a8-ad49-f988d9ede2dc">

``` you can click on copy to clipoboard option```
